src/rover/directory.js

```diff
@@ -0,0 +1,16 @@
+export const directory = {
+	NORTH: 0,
+	EAST: 1,
+	SOUTH: 2,
+	WEST: 3
+};
+
+export const Fig = 1;
```

Comment:
What is Fig?

Reply:
Figure

Reply:
Yeah, what is "figure"? What does it mean?